fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalTopNOperator.java

```diff
@@ -23,45 +24,55 @@ public class LogicalTopNOperator extends LogicalOperator {
     private final List<Ordering> orderByElements;
     private final long offset;
     private final SortPhase sortPhase;
+    private final TopNType topNType;
```

Comment:
I think this name is confusing, because the  ROW_NUMBER, RANK and DENSE RANK are actually wundow functions, not TopNType.

Reply:
Actually, it will truely effect the behavior of topn, I've considered `LimitType` and `TopNType`, and `TopNType` is choosed finally. Do you have any naming suggestion?